CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ way to update this template, but currently, we follow a pattern:
 
 ---
 ## Upcoming version 2018-08-XX
+* [change] Update the Sharetribe Flex SDK to the 1.0.0 version in NPM.
+  All the `sharetribe-sdk` imports are now using the new package name
+  `sharetribe-flex-sdk`.
```

Comment:
Also there's usually a link to the PR in the changelog entry.

Reply:
Order reversed and link added.